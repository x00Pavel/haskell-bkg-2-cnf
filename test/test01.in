S,A,B
a,b,c,d
S
S->AB
S->B
A->aAb
A->ab
B->A
A->S
B->cBd
B->cd