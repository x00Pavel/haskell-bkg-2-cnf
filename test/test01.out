S,A,B,c',d',a',b',<Bd>,<Ab>
a,b,c,d
S
S->AB
A->AB
B->AB
S->c'd'
c'->c
d'->d
S->a'b'
a'->a
b'->b
A->a'b'
A->c'd'
B->c'd'
B->a'b'
S->c'<Bd>
<Bd>->Bd'
S->a'<Ab>
<Ab>->Ab'
A->a'<Ab>
A->c'<Bd>
B->c'<Bd>
B->a'<Ab>